tests/phpunit/CRM/MembershipExtras/Service/MembershipPeriodType/FixedPeriodTypeMonthlyCalculatorTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use CRM_MembershipExtras_Service_MembershipPeriodType_AbstractBaseTest as BaseTest;
```

Comment:
Maybe we should send this class to `CRM_MembershipExtras_Test_Base_Service_...Xxxx`?

I've had to duplicate code in test classes, would be cool to have some place where we can put code we need for more than a single test class. Plus it's not actually a service, its a test.

Reply:
That's right, If in this case, we better remove it then. I don't want to send this to CRM_MembershipExtras_Test_Base_Service_* as it will make the test too complex which would make the test difficult to maintain and debug as well as we may introduce the bugs in the tests. 

Perhaps, in the future, we would some out with some nice traits to help us fabricate these test objects.